README.md

```diff
@@ -284,7 +284,7 @@ $ heroku container:release web
 This will make the application ready to run, but before it can run successfully, you'll need to set your configuration up. 
 You need to set your environment variables in heroku by going to https://dashboard.heroku.com/apps/$appName > Settings > Config Vars
 
-Once the configuration has been setup you can either run the app from https://dashboard.heroku.com/apps/$appName or from https://$appName.heroku.com
+Once the configuration has been setup you can either run the app from https://dashboard.heroku.com/apps/$appName or from https://$appName.herokuapp.com
```

Comment:
Rather than referring to it by `$appName` here, we should really add an actual link to your site! It can be puzzled out from your docker-compose & this but it would be much nicer to spell it out here 😄